README.md

```diff
@@ -8,7 +8,7 @@ Interface for running HUnit tests via Cabal 1.16.
 git clone ... && cd ... && cabal install
 ~~~
 
-Requires HUnit 1.2.* and Cabal 1.16.*
+Requires HUnit 1.2.* and Cabal >= 1.16
```

Comment:
Well, we don't know what future HUnit versions will be like so they might not be compatible, but I do know that Cabal 1.18 is compatible.